images/dcos/bootstrap.sh

```diff
@@ -163,6 +154,32 @@ EOF
   prepare_service_depends etcd-server ${etcd_server_list}/v2/stats/store getsSuccess ${deps}
 }
 
+prepare_etcd_proxy() {
+  prepare_service ${monitor_dir} ${service_dir} etcd-server ${ETCD_SERVER_RESPAWN_DELAY:-1} << EOF
+#!/bin/sh
+#TODO(jdef) don't run this as root
+#TODO(jdef) would be super-cool to have socket-activation here so that clients can connect before etcd is really ready
+exec 2>&1
+mkdir -p ${etcd_server_data}
```

Comment:
It's preparation of the environment. It does not change during the runtime. If it wasn't configurable it would even belong into the Dockerfile. But creating directories for persistent data does not belong into a service script for sure.